src/components/my-component/my-component.tsx

```diff
@@ -0,0 +1,32 @@
+import { Component, Prop, h } from '@stencil/core';
```

Comment:
Maybe this useless `my-component` component related files can be removed?

Reply:
Yes, I'll remove all the unnecessary files in a fixup.

Reply:
Fixed.